### ISSUE TITLE
ingest: include `url` field

### ISSUE THREAD

**Issue by joverlee521** [nextstrain/mpox#76]
### Context
See https://github.com/nextstrain/monkeypox/pull/72#issuecomment-1162442758

### Possible solution
GenBank urls can be specially added as `https://www.ncbi.nlm.nih.gov/nuccore/<genbank_accession>`
URLs for arbitrary non-GenBank sequences will have to be added through manual annotations. 

Related to https://github.com/nextstrain/pathogen-repo-guide/issues/76

**Comment by jameshadfield** [comment]
If `"accession": "AY741551"` is switched to `"genbank_accession": {"value": "AY741551"}` then auspice will automagically add the GenBank URL 😉

**Comment by joverlee521** [comment]
Yup, I was going through [this logic in Auspice](https://github.com/nextstrain/auspice/blob/9166cf304fc0d6f4da67bab94a3a63931fd819a1/src/components/tree/infoPanels/click.js#L55) and figured the accession/url combo would be best for monkeypox to support non-GenBank sequences.
